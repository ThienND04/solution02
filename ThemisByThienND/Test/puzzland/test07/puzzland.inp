7
9 27
I H Z D O Q R B A 
O A
H O
H Z
H D
I Q
D Q
I D
D O
O B
Z B
Z R
B A
Q R
I H
Z O
H A
I B
Z Q
D B
I Z
O Q
H B
O R
I R
Z A
Q B
Q A
9 21
D Z E P J X G Y S 
P X
E X
D J
Z Y
J Y
G Y
J X
D S
D E
X G
E J
Z X
Y S
X S
E G
Z G
J G
Z J
J S
P J
E S
5 9
A S X Y E 
Y E
S Y
A X
A Y
X E
S E
A E
X Y
A S
8 18
X K G N Q L E H 
X Q
K N
K E
K Q
N L
N H
X H
K G
Q L
X N
L E
E H
G N
N Q
X G
K H
G E
Q E
7 19
I B V W Z F T 
I Z
Z T
I B
B Z
W T
Z F
I V
I W
I F
B F
V F
V Z
W Z
W F
B V
F T
B W
V T
I T
7 14
J T B F Z X A 
J A
T X
F X
X A
B X
T F
F A
J F
B F
J B
B A
Z A
J X
B Z
6 9
J U H B D F 
H B
J U
D F
U F
H F
U D
U H
J B
J H
