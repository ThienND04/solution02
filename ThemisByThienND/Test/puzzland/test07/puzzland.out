IBDHOQRZA
DEGJPXZYS
ASYXE
XGEKQLNH
IBFVZWT
JFTXBZA
JBHUDF
